src/MissionManager/GeoFenceController.cc

```diff
@@ -602,3 +602,48 @@ bool GeoFenceController::isEmpty(void) const
     return _polygons.count() == 0 && _circles.count() == 0 && !_breachReturnPoint.isValid();
 
 }
+
+#ifdef CONFIG_UTM_ADAPTER
+void GeoFenceController::loadFlightPlanData()
+{
+    QJsonArray jsonPolygonArray;
+    QJsonDocument doc;
+    QList<QGeoCoordinate> geoCoordinates ;
+
+    for (int i=0; i<_polygons.count(); i++) {
```

Comment:
surround =/< of spaces. You can use Code->Reformat code  in clion, or do reformat code from your ide

Reply:
done